lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -601,7 +601,7 @@ def save_workflow_as(self, trans, workflow_name, workflow_data, workflow_annotat
             session.flush()
 
             try:
-                workflow, errors = workflow_contents_manager.update_workflow_from_dict(
+                workflow, errors = workflow_contents_manager.build_workflow_from_raw_description(
```

Comment:
Actually it was renamed to `update_workflow_from_raw_description` in https://github.com/galaxyproject/galaxy/pull/7019/files#diff-ec6f18e1ff51809344ca461c8971b3f6R352 :

```suggestion
                workflow, errors = workflow_contents_manager.update_workflow_from_raw_description(
```